contracts/TestMerkleLib.sol

```diff
@@ -34,7 +34,7 @@ contract TestMerkleLib {
         return MerkleLib.getProof(data, chunkSize, nChunkBits, chunkIdx);
     }
 
-    function getMaxLeafsNum(uint256 kvSize, uint256 chunkSize) public pure returns (uint256) {
-        return MerkleLib.getMaxLeafsNum(kvSize, chunkSize);
-    }
+    // function getMaxLeafsNum(uint256 kvSize, uint256 chunkSize) public pure returns (uint256) {
+    //     return MerkleLib.getMaxLeafsNum(kvSize, chunkSize);
+    // }
```

Comment:
Is there any particular reason to keep this function (even in comment)
Can it be removed entirely?